src/infinite-list/index.tsx

```diff
@@ -67,13 +68,17 @@ const InfiniteList = (props: InfiniteListProps, ref: any) => {
   const reloadPagesDebounce = useCallback(debounce(reloadPages, 500, {leading: false, trailing: true}), [reloadPages]);
 
   useEffect(() => {
+    if (props.onEndReached) {
```

Comment:
Why this change? Even if you pass 'onEndReached' callback you need to scroll the data in case it changes... this is not clear

Reply:
I don't want to scroll in the case of `onEndReached` I want to keep the scroll position in the same place (without this change, the view jump to the first item on any array update)
In case of new data changes, we have `scrollToIndex` on `InfiniteAgendaList`

Reply:
You want to block the scroll when onEndReached is called or implemented?

Reply:
If it exists

Reply:
But if the user wants to change the screen's background color every time the onEndReach is called, he needs to implement it, it doesn't mean he doesn't want the scroll to happen...

Reply:
Ok, I will add a new props on infinite List that will disable this scroll

Reply:
BTW, the infinite List does not supports onEndReached before this PR

Reply:
Yes, I noticed